### PR TITLE
Updated mcmc docs

### PR DESCRIPTION
**Pull request by Harry-Rich**
- Changed log_posterior function to use previously defined gp multivariate normal distribution as opposed to matrix method to improve clarity

- Changed some of the wording around posterior and liklihood for clarity, anything after the definition of the prior is the combination of the prior and the liklihood and so therefore the posterior? Let me know your thoughts

### CODE REVIEW

File: docs/source/methodology.ipynb [hunk 1]
```diff
@@ -426,9 +421,9 @@
    "source": [
     "def nll(*args) -> float:\n",
     "    \"\"\"\n",
-    "    General purpose negative log-likelihood.\n",
+    "    General purpose negative log-posterior.\n",
     "\n",
-    "    :return: Negative log-likelihood\n",
+    "    :return: Negative log-psoterior\n",
```
Comment (arm61) [inline docs/source/methodology.ipynb:426]: Typo
```suggestion
    "    :return: Negative log-posterior\n",
```

File: docs/source/methodology.ipynb [hunk 2]
```diff
@@ -39,7 +39,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 20,
```
Comment (arm61) [inline docs/source/methodology.ipynb:42]: Not sure why some of the execution counts are not null...

**Review by arm61 — APPROVED** [review]
Looks good. 

Some thoughts: 
- Any additional clarity improvement could be to split the log_prior and log_likelihood into two functions, up to you though. 
- Any thoughts on improvements to the companion text?

**Comment by Harry-Rich** [comment]
Split likelihood function and prior function. Haven't changed accompanying text yet wanted to have a conversation about it. Don't know why executions are not showing as null? I am clearing all outputs and resetting the kernel before commiting

File: docs/source/methodology.ipynb (hunk 1 above)
Comment (Harry-Rich) [reply docs/source/methodology.ipynb]: Fixed

**Comment by arm61** [comment]
I suggest we merge this and can look at the wording later.

**Pull request merged**